addOns/callhome/src/main/javahelp/org/zaproxy/addon/callhome/resources/help/contents/callhome.html

```diff
@@ -18,5 +18,20 @@ <H2>Check For Updates</H2>
 <H2>News</H2>
 The News call is used to find out the latest news item about ZAP - this is displayed in the ZAP desktop Quick Start panel. 
 
+<H2>Telemetry</H2>
+The Telemetry call reports the add-ons being used and some of the ZAP internal statistics.
+It will never collect details of the sites ZAP is being used on, details of the vulnerabilities or any personal identifiable information (PII).
+<p>
+For more background see the blog post <a href="https://www.zaproxy.org/blog/2021-10-25-zap-telemetry-plans/">ZAP Telemetry Plans</a>.
+
+<H2>Command Line option: -notel</H2>
+If the <code>-notel</code> is specified on the command line then no telemetry calls will be made. The other calls will not be affected.
```

Comment:
Drop "the" or make it "If the -notel switch"

Reply:
Done :)